werkzeug_encryptedcookie/__init__.py

```diff
@@ -47,14 +42,12 @@ def serialize(self, expires=None):
 
         if self.quote_base64:
             # bytes -> ascii bytes
-            string = base64.b64encode(string)
+            string = b''.join(base64.b64encode(string).splitlines()).strip()
```

Comment:
I have looked into how it's done in the original securecookie package here:

https://github.com/pallets/secure-cookie/blob/master/src/secure_cookie/securecookie.py#L204

But now I see that there is no need for that unless We understand why it has to be done that way. Since previous code has worked well for us before - I returned previous version.